pkg/services/ngalert/api/authorization.go

```diff
@@ -126,29 +126,42 @@ func (api *API) authorize(method, path string) web.Handler {
 	// convert/prometheus API paths
 	case http.MethodGet + "/api/convert/prometheus/config/v1/rules/{NamespaceTitle}/{Group}",
 		http.MethodGet + "/api/convert/prometheus/config/v1/rules/{NamespaceTitle}":
-		eval = ac.EvalAll(
-			ac.EvalPermission(ac.ActionAlertingRuleRead),
-			ac.EvalPermission(dashboards.ActionFoldersRead),
+		eval = ac.EvalAny(
+			ac.EvalPermission(ac.ActionAlertingProvisioningRead),
+			ac.EvalPermission(ac.ActionAlertingRulesProvisioningRead),
+			ac.EvalPermission(ac.ActionAlertingProvisioningReadSecrets),
+			ac.EvalAll(
+				ac.EvalPermission(ac.ActionAlertingRuleRead),
+				ac.EvalPermission(dashboards.ActionFoldersRead),
+			),
```

Comment:
```suggestion
		eval = ac.EvalAll(
				ac.EvalPermission(ac.ActionAlertingRuleRead),
				ac.EvalPermission(dashboards.ActionFoldersRead),
			)
```

Reply:
the provisioning actions are legacy. Let's not use it for new-ish APIs

Reply:
Updated: https://github.com/grafana/grafana/pull/100558/commits/4a4971ee0f2337bef97c211bfec2160db8ebb734